library/pk_wrap.c

```diff
@@ -34,7 +34,7 @@
 #include "mbedtls/ecp.h"
 #endif
 
-#if defined(MBEDTLS_ECDSA_C)
+#if defined(MBEDTLS_ECDSA_C) || defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
This change doesn't seem to be necessary anymore after the addition of the guards below. Am I missing something?